testsuite/performance/tests/pom.xml

```diff
@@ -48,11 +48,25 @@
         <keycloak.ds.max-pool-size>100</keycloak.ds.max-pool-size>
         <keycloak.ds.pool-prefill>true</keycloak.ds.pool-prefill>
         <keycloak.ds.ps-cache-size>100</keycloak.ds.ps-cache-size>
-        
+
         <keycloak-lb.jvm.memory>-Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m</keycloak-lb.jvm.memory>
         <keycloak-lb.http.max-connections>500</keycloak-lb.http.max-connections>
         <keycloak-lb.worker.io-threads>2</keycloak-lb.worker.io-threads>
         <keycloak-lb.worker.task-max-threads>16</keycloak-lb.worker.task-max-threads>
+
+        <docker.db.cpuset>0-1</docker.db.cpuset>
```

Comment:
Can we use the same naming scheme as for the other properties? `<SERVICE_NAME.docker.cpuset>` instead of `<docker.SERVICE_NAME.cpuset>`.

Reply:
Addressed